### DISCUSSION TITLE
SSD1306 or others?

### DISCUSSION THREAD

**Discussion by kidkill** [sethvoltz/OctoPrint-DisplayPanel#75] [Q&A]
Hey, I was wondering if I could use a SH1106 display with this? Also, will it work without the buttons, just the display?

**Comment by slipslyman** [comment]
I have been trying with the sh1106 type with the 7 pin (cs DC res mosi clk vcc grnd) without much luck. I followed the instructions for the Luma OLED driver install but so far multiple variations of failure. i2c detect will show an address at 3c or 3d depending how I wire it to the pi but that's about it for now.

**Reply by sethvoltz** [reply]
I don't have any of those displays to test myself and I'm not sure the Adafruit library used supports it anyway. 

If anyone gets it working, please let us know, and I'm happy to review PRs to add support.

**Reply by andrewerrington** [reply]
Just replying here so that people get a ping.

The Adafruit library does not support SH1106, but I have been experimenting with the luma.oled library which does. See my other thread.
